lms/static/sass/multicourse/_entitlement_dashboard.scss

```diff
@@ -0,0 +1,64 @@
+.wrapper-entitlement-action-more {
+    @extend .wrapper-action-more;
+
+    .entitlement-action {
+        @extend .action;
+    }
+
+    .entitlement-action-more {
+        @extend .action-more;
+    }
+}
+
+.entitlement-actions-dropdown {
+    @extend .actions-dropdown;
+
+    .entitlement-actions-dropdown-list {
+        @extend .actions-dropdown-list;
+
+        .entitlement-actions-item {
+            @extend .actions-item;
+
+            .entitlement-action {
+                @extend .action;
+            }
+        }
+    }
+}
+
+.entitlement-modal {
+    @extend .modal;
+
+    .entitlement-modal-inner-wrapper {
+        @extend .inner-wrapper;
+
+        .close-entitlement-modal {
+            @extend .close-modal;
+        }
+
+        .entitlement-unenroll-header {
+            @extend .unenroll-header;
+        }
+
+        .entitlement-modal-form-error {
+            @extend .modal-form-error;
+        }
+    }
+}
+
+.unenroll-entitlement-modal {
+    .wrapper-unenroll-entitlement-submit {
+        margin-bottom: 12px;
```

Comment:
$baseline*0.6 (any margin or padding should use $baseline, I believe).

Reply:
Changed.